contracts/script/deploy/certverifier/config/preprod.config.json

```diff
@@ -10,5 +10,5 @@
         "1_adversaryThreshold": 33
     },
 
-    "quorumNumbersRequired": "0x0001"
+    "quorumNumbersRequired": "0x02"
```

Comment:
Why quorum 2?

Reply:
was using it for deployments to populate onchain contexts for the correctness tests! can put back to 0 and 1!

Reply:
[0d0f7bb](https://github.com/Layr-Labs/eigenda/pull/1532/commits/0d0f7bbd771bccdab89ebb767b78dcd2163673bb)